test/e2e/scheduling/taint_based_evictions.go

```diff
@@ -123,10 +123,9 @@ var _ = SIGDescribe("TaintBasedEvictions [Serial]", func() {
 
 		ginkgo.By(fmt.Sprintf("Blocking traffic from node %s to the master", nodeName))
 		host, err := framework.GetNodeExternalIP(&node)
-		// TODO(Huang-Wei): make this case work for local provider
-		// if err != nil {
-		// 	host, err = framework.GetNodeInternalIP(&node)
-		// }
+		if err != nil {
```

Comment:
@ravisantoshgudimetla did you manage to make this e2e work on a local (or kind) cluster? I recalled that with a local provider, we can't ssh internalip:22 to manually disconnect the connection (which is key of this e2e). In other words, this test only works in gce/gke atm, and the comment should be uncommented out along with the logic of enabling ssh internalip:22.